src/core/server/saved_objects/migrations/next.ts

```diff
@@ -59,7 +59,7 @@ export type ResponseType<ControlState extends AllActionStates> = Awaited<
 export const nextActionMap = (client: ElasticsearchClient, transformRawDocs: TransformRawDocs) => {
   return {
     INIT: (state: InitState) =>
-      Actions.fetchIndices({ client, indices: [state.currentAlias, state.versionAlias] }),
+      Actions.initAction({ client, indices: [state.currentAlias, state.versionAlias] }),
```

Comment:
We should just update the readme with the new action name. It would also be good to write something in the readme about checking for cluster routing allocation cluster settings and why we fail the migration when it's disabled.

Reply:
Great catch!